pkgs/development/compilers/dmd/default.nix

```diff
@@ -24,7 +24,7 @@ stdenv.mkDerivation {
   # Buid and install are based on http://wiki.dlang.org/Building_DMD
   buildPhase = ''
       cd src/dmd
-      make -f posix.mak INSTALL_DIR=$out
+      make -f posix.mak INSTALL_DIR=$out AUTO_BOOTSTRAP=1
```

Comment:
In the travis build log I see

> curl -fsSL http://downloads.dlang.org/releases/2.x/2.068.2/dmd.2.068.2.linux.tar.xz | tar -C /tmp/.host_dmd-2.068.2 -Jxf - || rm -rf /tmp/.host_dmd-2.068.2

This will assuredly break in a sandboxed build.